app/js/router.js

```diff
@@ -1,6 +1,6 @@
 angular.module("app").config(function($routeProvider, $locationProvider) {
 
-  $locationProvider.html5Mode(true);
+  $locationProvider.html5Mode({enabled:true, requireBase: false});
```

Comment:
I think it would make more sense to add the required `<base href="/">` to `pages/index.us` instead of switching this off. :)